### PR TITLE
UCP/PROTO: Minor code cleanups in proto AM

### PR DESCRIPTION
**Pull request by dmitrygx**
## What

Minor code cleanups in proto AM

## Why ?

To align with the existing UCX code style

## How ?

1. Fix the name of the variable
2. Fix code alignments

### CODE REVIEW

File: src/ucp/proto/proto_am.inl [hunk 1]
```diff
@@ -413,10 +413,10 @@ ucp_proto_get_zcopy_threshold(const ucp_request_t *req,
                               const ucp_ep_msg_config_t *msg_config,
                               size_t count, size_t max_zcopy)
 {
-    ucp_worker_h     worker;
+    ucp_worker_h worker;
```
Comment (brminich) [inline src/ucp/proto/proto_am.inl:416]: afair, we have alignment by vars in many places..

Comment (dmitrygx) [reply src/ucp/proto/proto_am.inl:416]: yes, you're right. but it's more common to not have alignment by vars

**Pull request merged**